licenses/management/commands/publish.py

```diff
@@ -57,6 +61,35 @@ def add_arguments(self, parser: ArgumentParser):
     def _quiet(self, *args, **kwargs):
         pass
 
+    def output_text_files(self, *args, **kwargs):
+        output_dir = getattr(settings, "DISTILL_DIR", None)
+        licenses_dir = f"{output_dir}licenses/"
+        pattern = "legalcode*"
+        for root, dirs, files in os.walk(licenses_dir):
+            for filename in fnmatch.filter(files, pattern):
+                file_path = os.path.join(root, filename)
+                txt_filename = f"{filename}.txt"
+                txt_file_path = os.path.join(root, txt_filename)
+                with open(file_path, "r") as f:
+                    soup = BeautifulSoup(f, "lxml")
+                    plain_text_soup = soup.find(id="plain-text-marker")
+                    plain_text_html = plain_text_soup.prettify(formatter="html")
+                with tempfile.NamedTemporaryFile(mode="w+t") as temp:
+                    temp.write(plain_text_html)
+                    temp.seek(0)
```

Comment:
I don't think this seek is needed. pandoc will open the file separately and have its own file handle, which will be positioned initially at the beginning of the file.

Reply:
done